data/tlg0656/tlg002/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0656" xml:lang="grc" urn="urn:cts:greekLit:tlg0656.tlg002">
+    <ti:title xml:lang="lat">Euporista vel De simplicibus medicinis</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg0656.tlg002.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0656.tlg002">
+      <ti:label xml:lang="lat">De materia medica</ti:label>
```

Comment:
What is the basis for this label? I see Περί απλών φαρμάκων on the title page here.

Reply:
whoops, holdover from the metadata I was basing it off